modules/grpc/otel/otel-protobuf-parser.hpp

```diff
@@ -23,4 +23,35 @@
 #ifndef OTEL_PROTOBUF_PARSER_HPP
 #define OTEL_PROTOBUF_PARSER_HPP
 
+#include <grpcpp/support/config.h>
+
+#include "compat/cpp-start.h"
+#include "logmsg/logmsg.h"
+#include "compat/cpp-end.h"
+
+#include "opentelemetry/proto/resource/v1/resource.pb.h"
+#include "opentelemetry/proto/common/v1/common.pb.h"
+
+namespace syslogng
+{
+namespace grpc
+{
+namespace otel
+{
+namespace protobuf_parser
```

Comment:
Shouldn't this be a class instead?

We don't need to maintain a state currently, as I can see, but this may change even during a performance optimization.

Reply:
Yes, the `protobuf_parser` namespace is removed and it becomes a class in `otel: add skeleton of opentelemetry() parser` and `otel: implement parser`. Sorry for the messy history.

Reply:
Oops, sorry for the obsolete comment